joplin/base/models/__init__.py

```diff
@@ -0,0 +1,165 @@
+from django.db import models
```

Comment:
We should be able to clean out all of these imports once we get the unused stuff out of the models here